include/gridtools/stencil/frontend/run.hpp

```diff
@@ -140,6 +238,11 @@ namespace gridtools {
                 entry_point_t()(grid, data_store_map_t{fields...});
             }
 
+            template <class... Ts>
+            void run_impl(Ts...) {
+                static_assert(sizeof...(Ts) < 0, "Unexpected gridtools::stencil::run first argument.");
```

Comment:
```suggestion
                static_assert(sizeof...(Ts) < 0, "Unexpected first argument of gridtools::stencil::run.");
```

Reply:
Why first argument?

Reply:
`sizeof...(Ts) < 0` is a synonym of `false` here. Needed for technical reasons. This overload is fallback. It triggers when the right overload fails. I.e. when the first argument is not invocable with the right number of parameters.